.env.example

```
@@ -40,3 +37,12 @@ VITE_AUTH_ENABLED=false
 AUTH_SESSION_LIFETIME_SECONDS=43200
 AUTH_SESSION_ABSOLUTE_LIFETIME_SECONDS=86400
 AUTH_SESSION_REFRESH_THRESHOLD_SECONDS=1800
+
+# Discord interaction webhook ingress，本地默认关闭。
+DISCORD_INTERACTIONS_ENABLED=false
+DISCORD_INTERACTIONS_PLUGIN_ID=quantagent.official.source.discord_interaction_webhook
+DISCORD_INTERACTIONS_PUBLIC_KEY=
+DISCORD_INTERACTIONS_RESPONSE_TEXT=QuantAgent received your Discord interaction.
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)